app/controllers/renewal/WhatYouNeedController.scala

```diff
@@ -36,7 +36,7 @@ class WhatYouNeedController @Inject()(
                                        val ds: CommonPlayDependencies,
                                        renewalService: RenewalService,
                                        val cc: MessagesControllerComponents,
-                                       what_you_need: what_you_need)(implicit ec: ExecutionContext) extends AmlsBaseController(ds, cc) {
+                                       what_you_need: what_you_need) extends AmlsBaseController(ds, cc) {
```

Comment:
if we still need this ec implicit we might need to keep it removed in the base controller and just kept here